lib/internal/Magento/Framework/View/Layout/GeneratorPool.php

```diff
@@ -86,11 +87,17 @@ public function process(Reader\Context $readerContext, Generator\Context $genera
      * Add generators to pool
      *
      * @param GeneratorInterface[] $generators
+     *
      * @return void
      */
     protected function addGenerators(array $generators)
     {
         foreach ($generators as $generator) {
+            if (!($generator instanceof GeneratorInterface)) {
```

Comment:
Please get rid of unnecessary parentheses.

Reply:
Done.